### PR TITLE
Remove deprecated network param

### PR DESCRIPTION
**Pull request by bonomat**
Ticket was marked as stale so I thought I take the chance and create a quick PR before heading to bed :)

### CODE REVIEW

File: cnd/src/http_api/mod.rs [hunk 1]
```diff
@@ -289,7 +288,6 @@ impl From<bitcoin::Network> for BitcoinLedgerParams {
 #[derive(Clone, Copy, Debug, Serialize, Deserialize, PartialEq)]
 pub struct EthereumLedgerParams {
     chain_id: Option<ChainId>,
```
Comment (thomaseizinger) [inline cnd/src/http_api/mod.rs]: This is mandatory now and which could be enforced by removing the Option wrapper, that in turn also allows you to get rid of the TryFrom and the custom error :)

**Comment by bonomat** [comment]
> Please close this PR and create new PR to merge on dev


@D4nte : why not just change the target?

**Comment by thomaseizinger** [comment]
@bonomat I changed it for you

**Comment by D4nte** [comment]
> > Please close this PR and create new PR to merge on dev
> 
> @D4nte : why not just change the target? (one can just edit the PR and change master to dev)

Do as you wish.

**Pull request merged**